src/spi/cppthreads/CppThreadsTimer.cpp

```diff
@@ -66,3 +63,12 @@ bool CppThreadsTimer::stop() {
 bool CppThreadsTimer::isRunning() {
 	return this->started;
 }
+
+void CppThreadsTimer::routine()
+{
+	std::unique_lock<std::mutex> lock(this->cv_m);
+	this->cv.wait_for(lock, std::chrono::milliseconds(this->duration));
```

Comment:
Why was the predicate (3rd arg) removed from std::condition_variable::wait_for() ?

Reply:
The predicate was useless. started is always false when the condition is up.

Reply:
OK